src/Sulu/Bundle/MediaBundle/Media/FormatLoader/XmlFormatLoader11.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\MediaBundle\Media\FormatLoader;
+
+use Sulu\Bundle\MediaBundle\Media\FormatLoader\Exception\MissingScaleDimensionException;
+
+/**
+ * Class XmlFormatLoader for the version 1.1 of the image-formats.
+ */
+class XmlFormatLoader11 extends BaseXmlFormatLoader
+{
+    const SCHEMA_URI = 'http://schemas.sulu.io/media/formats-1.1.xsd';
+
+    const SCHEME_PATH = '/schema/formats/formats-1.1.xsd';
+
+    /**
+     * For a given format node returns the key of the format.
```

Comment:
Valid for all phpdocs in this format: use `{@inheritdoc}` if they are the same as in the base class.